src/utils/Membership.js

```diff
@@ -0,0 +1,53 @@
+const onKeyPress = event => {
+  const keyCode = event.keyCode || event.which;
+  const keyValue = String.fromCharCode(keyCode);
+  if (/\+|-|e/.test(keyValue)) event.preventDefault();
+};
+
+const isAmountValid = input => {
+  const reg = /^\s*(?=.*[1-9])\d*(?:\.\d{1,2})?\s*$/g;
+  const isValid = input * 1 > 0 && input * 1 <= 5000 && reg.test(input);
+  return isValid;
+};
+
+const getUrlParameter = name => {
+  const text = name.replace(/\[]/, '\\[').replace(/[\]]/, '\\]');
+  const regex = new RegExp(`[\\?&]${text}=([^&#]*)`);
+  const results = regex.exec(window.location.search);
+  const param =
+    results === null
+      ? 'generic'
+      : decodeURIComponent(results[1].replace(/\+/g, ' '));
+  return param;
+};
+
+const donate = (e, amount) => {
+  e.preventDefault();
+  const donateLink = 'https://donation.comicrelief.com/';
```

Comment:
Can we set this via a prop?

Also, would be nice to rename the function to handleSubmission or something more standard and identifiable